sotodlib/coords/helpers.py

```diff
@@ -321,36 +321,37 @@ def get_footprint(tod, wcs_kernel, dets=None, timestamps=None, boresight=None,
     fp1 = so3g.proj.FocalPlane.from_xieta(fake_dets, xieta1[0], xieta1[1])
 
     asm = so3g.proj.Assembly.attach(sight, fp1)
-    output = np.zeros((len(fake_dets), n_samp, 4))
+    planar = np.zeros((len(fake_dets), n_samp, 4))
```

Comment:
Why are there changes to `coords` in this PR?

Reply:
This is from the fix for the failing pmat test that was pushed a few days ago.  I should have done a merge commit to make it more clear.